cmake/modules/BuildDPDK.cmake

```diff
@@ -110,15 +110,22 @@ function(do_export_dpdk dpdk_dir)
 
   foreach(c
       bus_pci
+      bus_vdev
```

Comment:
are we mirroring https://github.com/scylladb/seastar/blob/809b3ef875e7edb7c56643812dc8347ff9a82cee/cmake/Finddpdk.cmake#L60-L85 ?

if yes, why do we, for instance, include the PMD driver for ixgbe but not e1000?

Reply:
Yes, except for PMD drivers. I only include drivers available in my environment which are ixgbe and i40e. Should we also mirror PMD drivers?

Reply:
Update: most of the PMD drivers are included now.